test/e2e/specs/editor/editor__post-advanced-flow.ts

```diff
@@ -38,19 +41,19 @@ describe( DataHelper.createSuiteTitle( `Editor: Advanced Post Flow` ), function
 	beforeAll( async function () {
 		page = await browser.newPage();
 
-		const testAccount = new TestAccount( accountName );
+		testAccount = new TestAccount( accountName );
 		await testAccount.authenticate( page );
-
-		postsPage = new PostsPage( page );
-		await postsPage.visit();
-		await postsPage.newPost();
-	} );
-
-	beforeAll( async function () {
-		editorPage = new EditorPage( page, { target: features.siteType } );
 	} );
 
 	describe( 'Publish post', function () {
+		beforeAll( async function () {
+			postsPage = new PostsPage( page );
+			await postsPage.visit();
+			await postsPage.newPost();
+
+			editorPage = new EditorPage( page, { target: features.siteType } );
+		} );
```

Comment:
**Idea (non-blocking):** I get the conceptual idea here, but what do you think of moving all these beforeAll blocks just each into their own `it` tests step? I just find that `it`s are much more graceful and descriptive when they fail (we can add a name to it, e.g.).

Reply:
@dpasque Yeah, when the hooks fail it is a bit of a mess. I'll give it some more thought, because one advantage of having it in a hook is that we don't end up padding the number of test steps.